client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+const Suggestions = require( './suggestions-mixin' );
```

Comment:
I'm working on eliminating the mixin as my next task. It was a carry-over from notifications-client. Thx!